docs/_assets/attributes.adoc

```diff
@@ -38,4 +38,7 @@
 :hashicorp-vault: https://developer.hashicorp.com/vault
 
 //AWS links
-:aws:  https://docs.aws.amazon.com/
\ No newline at end of file
+:aws:  https://docs.aws.amazon.com/
+
+// Apicurio links
+:apicurio-docs: https://www.apicur.io/registry/docs/apicurio-registry/2.6.x/
```

Comment:
Do we want to link to a specific version of the Apicurio docs? Should the version be parameterised?

Reply:
@PaulRMellor what approach would you normally take?

Reply:
Done.